file_organizer_v2/src/file_organizer/cli/profile.py

```
@@ -0,0 +1,509 @@
+@profile_command.command(name='activate')
+@click.argument('name')
+def activate_profile(name: str):
+    """Activate a profile (make it the current active profile)."""
+    try:
+        manager = get_profile_manager()
+
+        if manager.activate_profile(name):
+            click.echo(f"✓ Activated profile: {name}")
+        else:
+            click.echo(f"✗ Failed to activate profile '{name}'", err=True)
+            raise click.Abort() from e
```

Ruff (0.14.13)

Undefined name `e`

(F821)